### PR TITLE
helm modification

### PR DESCRIPTION
**Pull request by rahat-astro**
## 📋 Checklist

- [ ] The PR title is informative to the user experience

### CODE REVIEW

File: migrations/scripts/README.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ Or
 
 ```sh
 # helm 3
-helm3 list <release name> -n <namespace>
+helm list <release name> -n <namespace>
```
Comment (vishwas-astro) [inline migrations/scripts/README.md:68]: Do we still need the helm2 section above this?

File: bin/install-ci-tools [hunk 2]
```diff
@@ -26,23 +25,18 @@ else
...
 fi
 
-if [[ $FORCE -eq 0 && -f /tmp/bin/helm2 ]]; then
-  echo "helm2 is already installed."
-else
-  # Install helm
-  wget "https://get.helm.sh/helm-v${HELM2_VERSION}-${OS}-amd64.tar.gz"
-  tar -zxvf "./helm-v${HELM2_VERSION}-${OS}-amd64.tar.gz"
-  mv "${OS}-amd64/helm" /tmp/bin/helm2
+if [[ $FORCE -eq 0 && -f /tmp/bin/helm ]]; then
```
Comment (danielhoherd) [inline bin/install-ci-tools:28]: This `if` clause can be deleted.

File: migrations/scripts/README.md (hunk 1 above)
Comment (danielhoherd) [reply migrations/scripts/README.md:68]: Good catch, it should be deleted.

Comment (rahat-astro) [reply migrations/scripts/README.md:61]: My bad, there was a space in between "helm" and "2/3" so search I ran missed it! :( Thank you both for picking it up.

File: bin/install-ci-tools (hunk 2 above)
Comment (rahat-astro) [reply bin/install-ci-tools:28]: Thanks Daniel. done.

**Pull request closed without merging**